about/yocto-wpe.md

```diff
@@ -0,0 +1,112 @@
+---
+layout: post
+title: "Try WPE with Yocto images for Raspberry Pi"
+tags: [about]
+data: { dateless: "true" }
+permalink: /about/yocto-wpe.html
+---
+
+One way to easily test WPE is to use Yocto images for Raspberry Pi
+devices, write them in a micro-SD card and use them to boot the
+device.
+
+
+## Getting and flashing the image on a micro-SD card
+
+Use the following values:
+* **MACHINE**: one of **{raspberrypi3-mesa|raspberrypi4-64-mesa}**
+  * raspberrypi3-mesa is 32-bit, to be able to test WPE in a more resource-starved environment
+  * raspberrypi4-64-mesa is 64-bit
+* **SDIMG**: based on the machine, '`core-image-weston-browsers-${MACHINE}.wic`', e.g. '`core-image-weston-browsers-raspberrypi3-mesa.wic`'
+* **SDCARD_DEV**: whatever the device in your system, e.g. '`/dev/mmcblk0_REMOVE_THIS`'
+  * **note that PREVIOUS DATA WILL BE ERASED**, be specially careful to not overwrite the wrong disk/card
+
+The version of WPE in these images is **2.38.x**.
+
+
+### Getting the image
+
+Generically, using the substitutions mentioned previously:
+
+```
+wget https://wk-contrib.igalia.com/yocto/meta-webkit/browsers/stable/images/"${MACHINE}"/"${SDIMG}".{bz2,bmap}
+```
```

Comment:
This is a bit confusing... I understand how the sustitutions work, but couldn't we just provide the final URLs here so that users only have to click on them (or copy & paste the `wget` command) instead of having to manually do the substitutions first?

If we only have two images that would be only `wget` lines, right?

Reply:
Yes, initially we had more `wget`s because of the Wandboard images, but since Pablo suggested to remove them this part is probably more noise than signal.